bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/PlatformUI.java

```diff
@@ -268,18 +268,24 @@ public static IPreferenceStore createPreferenceStore(Class<?> clazz) {
 	 * interact with the plain E4 Application Model.</li>
 	 * </ol>
 	 *
-	 * The following example fails if run inside an E4 Application: <code>
+	 * The following example fails if run inside an E4 Application:
+	 * <p>
+	 * <code>
 	 *  PlatformUI.getWorkbench().getExtensionTracker()
-	 *  </code> and could be rewritten as: <code>
-	 *  PlatformUI.getApplication().map(MApplication::getContext).map(ctx->ctx.get(IExtensionTracker)).orElseThrow(PlatformUI.NO_WORKBENCH);
+	 * </code>
+	 * </p>
+	 * and could be rewritten as:
+	 * <p>
+	 * <code>
```

Comment:
Using {@code should allow to not escape special characters and thus direct copy of snippet from sources.

Reply:
Yep, I will change that